services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker_core.py

```diff
@@ -5,18 +5,24 @@
 from typing import Any
 
 import arrow
+from aiocache import cached
 from fastapi import FastAPI
+from models_library.projects import ProjectID
+from models_library.projects_nodes_io import NodeID
+from models_library.services import ServiceKey, ServiceVersion
 from prometheus_api_client import PrometheusConnect
 from pydantic import BaseModel
 from simcore_service_resource_usage_tracker.modules.prometheus import (
     get_prometheus_api_client,
 )
 
-from .models.resource_tracker_container import ContainerResourceUsage
+from .models.resource_tracker_container import ContainerScrapedResourceUsage
 from .modules.db.repositories.resource_tracker import ResourceTrackerRepository
+from .modules.db.repositories.user_and_project import UserAndProjectRepository
 
 _logger = logging.getLogger(__name__)
 
+_TTL = 300  # 5 minutes
```

Comment:
```suggestion
_TTL: Final[NonNegativeInt] = 300  # 5 minutes
```

Reply:
thanks, already preparing next PR with additional changes will include it there